ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -14,43 +14,36 @@
 
 LT:
   description: "Luminaire, lighting fixture."
+  is_abstract: true
   implements:
   - EQUIPMENT
-  uses:
-  - dimmer_percentage_command # in % indicating brightness
-
-LPS:
-  description: "Lighting photocell sensor."
-  implements:
-  - MTS
-
-LMS:
-  description: "Lighting motion sensor."
-  implements:
-  - MTS
 
 LTGW:
   description: "Lighting gateway."
-  implements:
-  - EQUIPMENT
-  
-LAC:
-  description: "lighting area controller."
+  is_abstract: true
   implements:
   - EQUIPMENT
 
 LCM:
   description: "Lighting control module."
+  is_abstract: true
```

Comment:
Can the description be clearer about differentiating between what an LCM and a LT is?

Reply:
ok